eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -119,66 +118,6 @@ function ResolvePkgJson($workFolder) {
   return ($pathsWithComplexity | Sort-Object -Property Complexity)[0].Path
 }
 
-# Parse out package publishing information given a .tgz npm artifact
-function ParseNPMPackage($pkg, $workingDirectory) {
```

Comment:
Why aren't you removing the other package functions?

Reply:
Am still working on this PR. I will sort it all out.